src/client.rs

```diff
@@ -182,6 +183,14 @@ impl Builder {
         self
     }
 
+    /// Set the first stream ID to something other than 1.
+    #[cfg(feature = "unstable")]
+    pub fn starting_stream_id(&mut self, stream_id: u32) -> &mut Self {
```

Comment:
A pretty dumb bikeshed... but maybe `initial` instead of `starting` since that's the lingo everything else uses.